sklearn/linear_model/ridge.py

```diff
@@ -1301,14 +1301,19 @@ class RidgeClassifierCV(LinearClassifierMixin, _BaseRidgeCV):
         weights inversely proportional to class frequencies in the input data
         as ``n_samples / (n_classes * np.bincount(y))``
 
+    store_cv_values : boolean, default=False
+        Flag indicating if the cross-validation values corresponding to
+        each alpha should be stored in the `cv_values_` attribute (see
+        below). This flag is only compatible with `cv=None` (i.e. using
```

Comment:
Double backticks

Reply:
ok! Dind't touch this, I copied it from RidgeCV. Should I change documentation for store_cv_values_ and cv_values_ too in other parts of the file to keep consistency??

Reply:
I suppose so